src/main/resources/templates/addRemoveQuestions.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<body>
+<h2>Add/Remove Questions</h2>
+<p th:text="${survey.name}"></p>
+
+<!-- Form to add a new question -->
+<form th:action="@{/addQuestion/{surveyId}(surveyId=${survey.id})}" method="post">
+    <label for="questionText">New Question:</label>
+    <input type="text" id="questionText" name="questionText" required>
+    <button type="submit">Add Question</button>
+</form>
+
+<!-- Display existing questions and allow deletion -->
+<ul>
+    <li th:each="question : ${survey.questions}">
+        <span th:text="${question.question}"></span>
+        <a th:href="@{'/deleteQuestion/' + ${survey.id} + '/' + ${question.id}}">Delete</a>
+    </li>
+</ul>
+<!-- Add other content or actions as needed -->
+
+<a th:href="@{/saveSurvey}" class="btn btn-primary">Save Survey</a>
```

Comment:
No point to this button, adding and deleting questions already saves the changes.